guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/wherefore-art-thou/index.md

```diff
@@ -81,31 +81,29 @@ Check equivalence of `Object` in `collection` with `Object` passed as second par
       // "What's in a name? that which we call a rose
       // By any other name would smell as sweet.”
       // -- by William Shakespeare, Romeo and Juliet
-      var srcKeys = Object.keys(source);
 
-      return collection.filter(function (obj) {
-        return srcKeys.every(function (key) {
-          return obj.hasOwnProperty(key) && obj[key] === source[key];
-        });
-      });
+      return collection.filter(obj =>
+        Object.keys(source).every(key =>
+          obj[key] === source[key]
+        )
+      );
     }
 
     // test here
     whatIsInAName([{ first: "Romeo", last: "Montague" }, { first: "Mercutio", last: null }, { first: "Tybalt", last: "Capulet" }], { last: "Capulet" });
 
-![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLmi/0' target='_blank' rel='nofollow'>Run Code</a>
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/repls/WholeCylindricalTransformations' target='_blank' rel='nofollow'>Run Code</a>
```

Comment:
We are now removing links to repl.it and Codepen, so just remove this entire line.

Reply:
I'm removing it for the OP since we haven't heard back in 20 days. @tmshkr I hope that's fine with you